sema/check/testing/base.py

```
@@ -0,0 +1,31 @@
+
+@dataclass
+class TestResult:
+    success: bool
+    error: bool
+    message: str
+    url: str
+    type_test: str
+
+
+class TestBase(ABC):
+    def __init__(self, url: str, type_test: str, options: dict):
```

Ruff

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)